### DISCUSSION TITLE
Allow Concurrent Peer Connections w/ same UID

### DISCUSSION THREAD

**Discussion by xizhao** [peers/peerjs#1233] [Ideas & Feature requests]
Allow multiple instances of the same peer (id) to be connected concurrently.

Note: PeerJS will have to be modified as well to multicast messages to concurrent peers.

**Comment by xizhao** [comment]
Notice: push was an in-progress and untested.

**Comment by michelle** [comment]
I don't quite understand what this is (re: 'concurrent'). You can already use a Peer to connect to multiple other Peers, or even create multiple Peers per client.

**Comment by xizhao** [comment]
per UID

**Comment by koko-ng** [comment]
It can be useful for a chat, when you send a message to somebody the message is deliverd to every devices (i.e. a mobile phone, a computer, a tablet). The client (or the server) doesn't need to find the good UID.

**Comment by panurge-ws** [comment]
+1

**Comment by dlinx** [comment]
+1
